deployment/linux-capability.md

```diff
@@ -84,6 +84,21 @@ Inheritable: dac_override, dac_read_search
 Permitted:   dac_override, dac_read_search
 ```
 
+#### Note: using CAP\_DAC\_READ\_SEARCH in docker container
+
+When using `CAP_DAC_READ_SEARCH` in a Docker container, you need to add the `--cap-add DAC_READ_SEARCH` option to the `docker run` command.
+Or, if you are using `docker compose`, you need to add `cap_add` to the service definition.
```

Comment:
It might be better to explain "the context" - such as collecting logs with non-root user.